lighthouse-core/report/v2/renderer/category-renderer.js

```diff
@@ -167,6 +176,69 @@ class CategoryRenderer {
     element.appendChild(passedElem);
     return element;
   }
+
+  _renderAuditGroup(audits, tag) {
+      const auditGroupElem = this._dom.createElement('details', 'lh-audit-group');
+      const auditGroupHeader = this._dom.createElement('div', 'lh-audit-group__header');
+      auditGroupHeader.textContent = tag.title;
+
+      const auditGroupDescription = this._dom.createElement('div', 'lh-audit-group__description');
```

Comment:
you can style the details arrow like the rest by adding a selector like https://github.com/GoogleChrome/lighthouse/blob/2ae69be650e4a8e303ac847b1fe4b076132ac567/lighthouse-core/report/v2/report-styles.css#L228-L234 and generating markup like https://github.com/GoogleChrome/lighthouse/blob/2ae69be650e4a8e303ac847b1fe4b076132ac567/lighthouse-core/report/v2/templates.html#L19-L22. `lh-score__arrow` is the gem.